src/framework/application.js

```diff
@@ -1103,6 +1103,7 @@ class Application extends EventHandler {
         this.fire('initialize');
 
         this.systems.fire('postInitialize', this.root);
+        this.systems.fire('initializeEntityReferences', this.root);
         this.fire('postinitialize');
```

Comment:
We should remove this event as the engine nor editor listens for it. Cautious of removing this on a patch release